pkg/cmd/dev/doctor.go

```diff
@@ -279,6 +279,12 @@ Make sure one of the following lines is in the file %s/.bazelrc.user:
 			if !d.checkLinePresenceInBazelRcUser(cfg.workspace, "build:engflow --jobs=200") {
 				return fmt.Sprintf("Make sure the following line is in %s/.bazelrc.user: build:engflow --jobs=200", cfg.workspace)
 			}
+			if !d.checkLinePresenceInBazelRcUser(cfg.workspace, "build:engflow --credential_helper=mesolite.cluster.engflow.com=engflow_auth") {
```

Comment:
Nit: Lines 279-287 can be refactored with a loop, optional if you want to do this.